pkg/crc/machine/start.go

```diff
@@ -469,7 +469,7 @@ func (client *client) Start(ctx context.Context, startConfig types.StartConfig)
 
 	logging.Info("Adding crc-admin and crc-developer contexts to kubeconfig...")
 	if err := writeKubeconfig(instanceIP, clusterConfig); err != nil {
-		logging.Errorf("Cannot update kubeconfig: %v", err)
+		return nil, err
```

Comment:
@guillaumerose Doesn't this change will impact the ux side of message because till now these are `Error` messages but at the end user have the message that cluster is unstable but still print the accessible point which means cluster might reconcile and works properly, With this change user might have to delete and start.

```
The server is accessible via web console at:
  https://console-openshift-console.apps-crc.testing

Log in as administrator:
  Username: kubeadmin
  Password: gpxkx-4kox9-4dJvw-c4CBs

Log in as user:
  Username: developer
  Password: developer

Use the 'oc' command line interface:
  $ eval $(crc oc-env)
  $ oc login -u developer https://api.crc.testing:6443
```